src/utils/include/utils/index.hpp

```diff
@@ -58,28 +58,51 @@ inline void unravel_index(InputIterator dimensions_begin,
   }
 }
 
+namespace MemoryOrder {
+constexpr struct RowMajor {
+} row_major;
+constexpr struct ColumnMajor { } coulumn_major; } // namespace MemoryOrder
```

Comment:
typo + formatting
```suggestion
constexpr struct ColumnMajor {
} column_major;
} // namespace MemoryOrder
```

Reply:
Actually this is what clang-format wants